src/cpl/nuopc/atm_comp_nuopc.F90

```diff
@@ -1,2326 +1,2100 @@
+   use ESMF                , only : ESMF_Time, ESMF_TimeGet
+   use ESMF                , only : ESMF_Alarm, ESMF_ClockGetAlarm, ESMF_AlarmRingerOff, ESMF_AlarmIsRinging
+   use ESMF                , only : ESMF_ClockGetAlarmList, ESMF_ALARMLIST_ALL, ESMF_AlarmSet
+   use ESMF                , only : ESMF_TimeInterval, ESMF_TimeIntervalGet
+   use ESMF                , only : ESMF_CalKind_Flag, ESMF_MAXSTR, ESMF_KIND_I8
+   use ESMF                , only : ESMF_CALKIND_NOLEAP, ESMF_CALKIND_GREGORIAN
+   use ESMF                , only : ESMF_GridCompSetEntryPoint
+   use ESMF                , only : ESMF_VM, ESMF_VMGetCurrent, ESMF_VMGet
+   use ESMF                , only : ESMF_LOGMSG_INFO, ESMF_LOGERR_PASSTHRU
+   use ESMF                , only : ESMF_LogWrite, ESMF_LogSetError, ESMF_LogFoundError
+   use ESMF                , only : ESMF_SUCCESS, ESMF_METHOD_INITIALIZE, ESMF_FAILURE, ESMF_RC_NOT_VALID
+   use ESMF                , only : ESMF_STAGGERLOC_CENTER, ESMF_STAGGERLOC_CORNER
+   use NUOPC               , only : NUOPC_CompDerive, NUOPC_CompSetEntryPoint, NUOPC_CompSpecialize
+   use NUOPC               , only : NUOPC_CompFilterPhaseMap, NUOPC_IsUpdated, NUOPC_IsAtTime
+   use NUOPC               , only : NUOPC_CompAttributeGet, NUOPC_Advertise
+   use NUOPC               , only : NUOPC_SetAttribute, NUOPC_CompAttributeGet, NUOPC_CompAttributeSet
+   use NUOPC_Model         , only : model_routine_SS           => SetServices
+   use NUOPC_Model         , only : SetVM
+   use NUOPC_Model         , only : model_label_Advance        => label_Advance
+   use NUOPC_Model         , only : model_label_DataInitialize => label_DataInitialize
+   use NUOPC_Model         , only : model_label_SetRunClock    => label_SetRunClock
+   use NUOPC_Model         , only : model_label_Finalize       => label_Finalize
+   use NUOPC_Model         , only : NUOPC_ModelGet
+   use shr_kind_mod        , only : r8=>shr_kind_r8, i8=>shr_kind_i8, cl=>shr_kind_cl, cs=>shr_kind_cs
+   use shr_sys_mod         , only : shr_sys_abort
+   use shr_log_mod         , only : shr_log_getlogunit, shr_log_setlogunit
+   use shr_cal_mod         , only : shr_cal_noleap, shr_cal_gregorian, shr_cal_ymd2date
+   use shr_const_mod       , only : shr_const_pi
+   use shr_orb_mod         , only : shr_orb_decl, shr_orb_params, SHR_ORB_UNDEF_REAL, SHR_ORB_UNDEF_INT
+   use cam_instance        , only : cam_instance_init, inst_suffix, inst_index
+   use cam_comp            , only : cam_init, cam_run1, cam_run2, cam_run3, cam_run4, cam_final
+   use cam_comp            , only : cam_timestep_init, cam_timestep_final
+   use camsrfexch          , only : cam_out_t, cam_in_t
+!   use radiation           , only : nextsw_cday  !uncomment once radiation has been CCPP-ized -JN
+   use cam_logfile         , only : cam_set_log_unit, iulog
+   use cam_abortutils      , only : check_allocate
+   use spmd_utils          , only : spmd_init, masterproc, iam, mpicom
```

Comment:
the nitpickiest of nitpicks - i think you can remove "mpicom" from this use statement

Reply:
Good catch (or I guess I should say nitpick)!  Done!